agent/src/main/java/com/teamscale/jacoco/agent/AgentBase.java

```diff
@@ -55,7 +55,10 @@ public AgentBase(AgentOptions options) throws IllegalStateException {
 		}
 
 		logger.info("Starting JaCoCo agent for process {} with options: {}",
-				ManagementFactory.getRuntimeMXBean().getName(), options.getOriginalOptionsString());
+				ManagementFactory.getRuntimeMXBean().getName(),
+					options.shouldObfuscateSecurityRelatedOutputs() ?
+					options.getObfuscatedOptionsString() :
+							options.getOriginalOptionsString());
```

Comment:
I'm afraid Teamscale is right here. This is not considered "simple usage" of the ternary operator, so you have to use if..else.

Reply:
I have moved it to a separate function. Thanks.